rfcs/0000_future_work.md

```diff
@@ -0,0 +1,167 @@
+- Start Date: 2025-08-14
+- RFC PR: (leave this empty, it will be filled in after RFC is merged)
+
+
+# Summary
+
+This RFC proposes to create a centralized process to update the future work artifacts.
+
+# Motivation
+
+Currently, there are three places to update the future work. 
+- The markdown files in `docs/future-work` which holds the detailed descriptions of the work being suggested.
+- The Google sheet where tasks are broken up by tag and type, as well as indicating who is currently working on a task.
+- Our internal task tracking system, Shortcut.
+
+There are some downsides to this:
+- Updating 3 places when work is started, and maintaining those places as the work progresses.
+- There is currently no way to move from viewing the spreadsheet back to the detailed descriptions of that work.
+- It's cumbersome to add new structured data and have it be useful to the community.
+- It is not possible to filter the tasks by different aspects. The only grouping we currently have is which Monty component a task relates to (columns in the sheet + sections in the docs). But it would be useful to also be able to filter by skills required, scope, output type, etc...
+
+There are some upsides to the proposed approach:
+
+- We would like the community to more easily find work to contribute to based on their skills, desire and available time. The skills and scope data doesn't currently exist and adding it to a single place would be preferable.
+
```

Comment:
This blank line will add extra padding between these two list items. Unless that's deliberate, it should probably be removed. Initially, I thought it would create two separate unordered lists in the HTML, but it doesn't look like the the GitHub renderer does.

```suggestion
```